app/src/components/SearchBar.css

```diff
@@ -0,0 +1,8 @@
+input {
```

Comment:
Could we use styled-components to keep the styling solution consistent? See examples in util.tsx